src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -217,7 +457,7 @@ private static Suppression getNearestSuppression(List<Suppression> suppressions,
     }
 
     /** The class which represents the suppression. */
-    public static class Suppression {
+    /* package */ static class Suppression {
```

Comment:
Why did this change?

Reply:
To hide the implementation details. But this should be `private`. I'll start an issue for this.